storagesystem/types.go

```diff
@@ -284,6 +285,9 @@ func init() {
 
 			for _, provider := range providers {
 				option := option.Copy()
+				if backend.Name == "local" && option.Name == "encoding" {
+					option.Default = encoder.Base
```

Comment:
I suppose you encounter this after some debugging. Could you tell me more how this becomes necessary?

Reply:
Mac doesn't set it to encoder.Base. That's the issue. I work on a Mac. It's impossible otherwise for me to run generate without changing the code, which then means it won't pass CI

Reply:
while you're on mac, can you try `GOOS=linux go generate ./...`

Reply:
I am thinking of, maybe we should move doc gen outside of go generate and then have a github workflow that generates and commit to the repo upon manual dispatch. What do you think?